xio-core/src/main/java/com/xjeffrose/xio/http/StreamingResponseData.java

```diff
@@ -5,6 +5,7 @@
 import io.netty.handler.codec.http.HttpResponseStatus;
 import lombok.ToString;
 
+/** Interface representing a partial HTTP1/2 Request */
```

Comment:
Let's say segmented instead of partial

copy paste error: Request should be Response